src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -33,6 +33,7 @@ public class ConnectionService
         public const string AdminConnectionPrefix = "ADMIN:";
         internal const string PasswordPlaceholder = "******";
         private const string SqlAzureEdition = "SQL Azure";
+        public const int maxTolerance = 2 * 60; // two minutes

```

Comment:
1. for const, you should follow Pascal casing just like other consts here. first letter to be upper case
2. is there a reason you choose 2 minutes? add it to the comment.